keycloak-event-emitter/src/main/java/io/cloudtrust/keycloak/eventemitter/CompleteEventUtils.java

```diff
@@ -0,0 +1,73 @@
+package io.cloudtrust.keycloak.eventemitter;
+
+import io.cloudtrust.keycloak.eventemitter.customevent.ExtendedAdminEvent;
+import io.cloudtrust.keycloak.eventemitter.customevent.ExtendedAuthDetails;
+import io.cloudtrust.keycloak.eventemitter.customevent.IdentifiedAdminEvent;
+import org.apache.commons.lang3.StringUtils;
+import org.keycloak.events.Details;
+import org.keycloak.events.Event;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.RealmModel;
+import org.keycloak.models.UserModel;
+
+import java.util.HashMap;
+import java.util.function.Consumer;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Utility class to complete attributes of Events an AdminEvents.
+ */
+public class CompleteEventUtils {
```

Comment:
It could be a collection of static methods. It is what we often do when doing utils classes

Reply:
Yeah ok I'll change it for consistency. I thought about doing this since all methods require a KeycloakSession, but it is better for helper functions to stay static